webpack/assets/javascripts/react_app/components/DiffView/DiffConsts.js

```diff
@@ -1,2 +1,8 @@
 export const SPLIT = 'split';
 export const UNIFIED = 'unified';
+
+// For tests
```

Comment:
if its only for tests it should be in the test file, or in a `fixtures.js` file in this folder

Reply:
@MariaAga do you think we should put all the fixtures of DiffView & Diffcontainer in a single fixtures file called: Diff.fixtures.js ?  
Or we should rather have two seperate files the first called DiffView.fixtures.js and the second called DiffContainer.fixtures.js ? while each one contains the fixtures relevant only for it
(I used a fixture called `diffMock` defined in DiffView.fixtures.js in the test of DiffContainer that is why I thought about having one file containing all of the fixtures of the DIff component)

Reply:
Yes, you can put all fixtures related to the `DiffView` folder in one file.